backend/core/urls.py

```diff
@@ -1,7 +1,8 @@
-from django.conf.urls import url
+from django.urls import path
 from . import views
 
+app_name = 'core'
```

Comment:
Why do we need this `app_name` here?

Reply:
I seen this on the doc of Django==2.0: https://docs.djangoproject.com/en/2.0/topics/http/urls/#url-namespaces-and-included-urlconfs